src/index.js

```diff
@@ -308,20 +308,20 @@ class WarmUp {
     /** Log functions being warmed up */
     functions.forEach((func) => this.serverless.cli.log(`WarmUp: ${func.name}`));
 
-    const warmUpFunction = `"use strict";
+    const warmUpFunction = `'use strict';
 
 /** Generated by Serverless WarmUp Plugin at ${new Date().toISOString()} */
-const aws = require("aws-sdk");
-aws.config.region = "${region}";
+const aws = require('aws-sdk');
+aws.config.region = '${region}';
 const lambda = new aws.Lambda();
 const functions = ${JSON.stringify(functions)};
 
 module.exports.warmUp = async (event, context) => {
-  console.log("Warm Up Start");
+  console.log('Warm Up Start');
 
   const invokes = await Promise.all(functions.map(async (func) => {
     let concurrency;
-    const functionConcurrency = process.env["WARMUP_CONCURRENCY_" + func.name.toUpperCase().replace(/-/g, '_')]
+    const functionConcurrency = process.env[\`WARMUP_CONCURRENCY_${func.name.toUpperCase().replace(/-/g, '_')}\`]
```

Comment:
This line still fails ESLint with `error  'func' is not defined  no-undef`, could be bypassed by using the eslint comment `// eslint-disable-line no-undef`